tpu_inference/layers/jax/attention/gpt_oss_attention.py

```diff
@@ -165,6 +179,9 @@ def _ragged_paged_attention_wrapper(*args):
                 *args,
```

Comment:
After this PR, do you need to support fp8 kv cache quantization in the pallas kernel?